src/app/public/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -192,6 +198,10 @@ export class SkyColorpickerInputDirective
         this.skyColorpickerInput.initialColor = value;
       }
       this.skyColorpickerInput.lastAppliedColor = value;
+      let control: FormControl = (<NgControl>this.inj.get(NgControl)).control as FormControl;
+      if (control) {
```

Comment:
Could you use something simpler to get the FormControl, like this?
```
constructor(
  @Optional() private formControl: NgControl
) { }

if (this.formControl) {
  // ...
}
```

Reply:
I'll try. I don't believe so based on what I was finding online as I researched but I might be wrong

Reply:
Good deal. Thanks for checking.